service_scripts/buildbot_master_all_restart.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Please create buildbot sub dir and rename scripts like stop_master.sh, restart_worker.sh